CODEOWNERS

```diff
@@ -91,6 +91,8 @@ extensions/filters/common/original_src @snowp @klarose
 /*/extensions/filters/listener/http_inspector @yxue @lizan
 # attribute context
 /*/extensions/filters/common/expr @kyessenov @yangminzhu @lizan
+# language
+/*/extensions/filters/http/language @diceride @mattklein123
```

Comment:
Have you coordinated with @mattklein123 regarding sponsorship? If not, then you either need to have maintainer sponsor or move the filter code into the `contrib` directory, per the contribution policy.

Reply:
I did not know how to proceed here as at least two contributors are required. For now, I added you @yanavlasov to the contrib section of the CODEOWNERS file. Please let me know if this is all right with you. Otherwise I don't know how to proceed here.

```txt
/contrib/language/ @diceride @yanavlasov
```

Reply:
I did not know whom to add as at least two contributors are required. For now, I added you @yanavlasov to the contrib section of the CODEOWNERS file. Please let me know if this is all right with you. Otherwise I don't know how to proceed here.

```txt
/contrib/language/ @diceride @yanavlasov
```